doc/sphinxext/redirect_from.py

```diff
@@ -49,6 +49,7 @@ def setup(app):
     RedirectFrom.app = app
     app.add_directive("redirect-from", RedirectFrom)
     app.connect("build-finished", _generate_redirects)
+    return {'parallel_read_safe': True}
```

Comment:
Why is this change here?

Reply:
Sorry, this is my personal temporary workaround to have parallel local doc-build. (The correct way to get parallel doc builds back is #19686, but that needs still in a bit of work).